contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Code Review Guidelines"
+weight: 1
+slug: "reviewer-guidelines" 
+---
+
+*This Section is new and in progress. Expect this document to change often*
+
+**Table Of Contents**
+
+- [Reviewer Guidelines](#reviewer-guidelines)
+    + [Reviewing Process](#reviewing-process)
+    + [Reviewing Ideology](#reviewing-ideology)
+    + [Something To Avoid](#something-to-avoid)
+    + [Additional Readings](#additional-readings)
+
+## Reviewing Process
+
+These are some general guidelines for review, based on the size or type of pull request. 
+
+Note that each reviewer has their own style, adapted over time, so don't feel you need to explicitly commit to every one of these guidelines.
+
+### Flakes/Test Fixes
+- If the PR is to fix any test in the repository's CI , you can easily review the change by seeing the `Green Ticks` in the PR Checks.
+
+### Very Minor Changes (like typo fixes)
+
+If the PR makes typo/grammar fixes or comprises a very minor change in a repository, you can fairly easily review the PR by using the GitHub UI. You should ensure that the PR passes all GitHub checks before approval.
+
+### Minor Changes
+
+If the PR is a minor change in the repository, you can browse through the changes line of codes through the GitHub UI. You can use the ["Split Diff View"](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-comparing-branches-in-pull-requests) for easy view of each LOC and also you can see changes made in the each commit of the patch using the UI.
+
+### Major Change:
+- If the PR is a major change which may affect a lot of section of your repository, You can pull the local changes of the PR into your local system and browse through each and every change made in the PR. You can also test & play around with the changes done in the PR to ensure each and every change is as expected.
+- To fetch the Pull Request locally, Use this -
+```sh
+$ git checkout -b `<pr-to-review>`
+$ git pull upstream/<PULL ID>/head
+```
+
+## Reviewing Ideology
+
+### Deciding what to review ?
+- As a Maintainer, you will might be completely overloaded with the Pending PRs left to review. Some of which will be from experienced contributors and some from the new contributor.
+- Since you know the experienced contributor already knows the guidelines for PR and Code to be written for repository. You should review the PR and patches from the experienced contributors first, as it will be less time consuming.
```

Comment:
```suggestion
- Since you know the experienced contributor already knows the guidelines for PR and Code to be written for repository, you should review the PR and patches from the experienced contributors first, as it will be less time consuming.
```